<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsChangeWidget</class>
 <widget class="QWidget" name="SettingsChangeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="page"/>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="FMWidget" name="fmWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="NoiseWidget" name="noiseWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="SettingsChangeFooterWidget" name="footerWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>84</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>84</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FMWidget</class>
   <extends>QWidget</extends>
   <header>fmwidget/fmwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NoiseWidget</class>
   <extends>QWidget</extends>
   <header>noisewidget/noisewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SettingsChangeFooterWidget</class>
   <extends>QWidget</extends>
   <header>pianorollwidget/settingschangefooterwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
